docs/project/internal.md

```diff
@@ -0,0 +1,364 @@
+
+Internal
```

Comment:
You said "I skipped over the doc on internals which are by definition not open to on-chain interactions" and I think you have right to skip. 

But, in this case, they should not be appear on the webpage Gitbook ? As you did with storages. 
https://carbonable.gitbook.io/starknet-smart-contracts/starknet-smart-contracts/index/internal